data/changelog.txt

```diff
@@ -122,33 +77,8 @@ Display missing and obsolete mods for current world.
 Require grenades and other explosives to be wielded before activation.
 Add bigger mini-maps to all sidebar layouts.
 Add keycode support to uilist.
-Allow disabling robots by 'e'xamining them
-rename wander spawns to wandering hordes
-Show correct duration for permanent spells in spell info
-Warn if friendly creature or non-hostile NPC enters line of fire
-Add page up and page down keybindings to ascend and descend stairs
-Crafting GUI - Show nearby + produced amounts
-Show electric power in vehicle part descriptions
-Warn the player if they're about to set their firewood fuel source ablaze.
-Add sweet cataclysm sprites to BrownLikeBears
-Make item categorisation consistently use contents where appropriate
-Add preview to gun modification
-Add debug menu option to display transparency cache
-Add demon spiders and black wyrms to BLB
-Highlight selection's parent container and contents in the inventory screen
-Highlight selection's parent container and contents in the multidrop menu screen
-Add help for what higher numbers mean in world options
```

Comment:
This is one that would/has caused me to investigate the matter further; I'd keep it.

Reply:
The thing is, for all of these "changelog as documentation" comments, the changelog is not that kind of documentation, the criteria is, "when we announce the release, what are the most exciting/notable changes".  In aggregate some of the infrastructure stuff that's not user-facing makes the cut because of impact, but specific entries about adding documentation definitely do not.  Specific bugfixes make the cut because of impact, not for reference value.